language/tools/move-mv-llvm-compiler/tests/dwarf-tests.rs

```diff
@@ -32,6 +32,24 @@
 //! directort basic-coin-build is created.
 //! - Run `move-mv-llvm-compiler` with -g option. This will create *.dbg_info files.
 //! - Compare the dbg_info.actual files with dbg_info.expected files.
+//!
+//! To debug in VS Code create this config:
+//!         {
+//!            "type": "lldb",
+//!            "request": "launch",
+//!            "name": "dwarf-tests direct call",
+//!            "env": {
+//!                    "RUST_BACKTRACE": "all",
+//!                    "RUST_LOG": "debug",
+//!                    "CARGO_MANIFEST_DIR": "something like /home/sol/work/git/move/language/tools/move-mv-llvm-compiler",
+//!                    "LLVM_SYS_150_PREFIX": "something like /home/sol/work/git/platform-tools/out/rust/build/x86_64-unknown-linux-gnu/llvm",
+//!                    "PLATFORM_TOOLS_ROOT": "something like /home/sol/work/git/platform-tools/out/deploy"
+//!            },
+//!            "program": "something like /home/sol/work/git/move/target/debug/deps/dwarf_tests-XXXXXXXXXXXXXXXX",
+//!            "args": ["--test"],
+//!            "cwd": "something like /home/sol/work/git/move/language/tools/move-mv-llvm-compiler",
+//!            "stopOnEntry": false
+//!        }
```

Comment:
This shouldn't be in the code as a comment. If you want to keep this information, add it to a doc file, for example in language/tools/move-mv-llvm-compiler/docs/Development.md